src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Default {
+    private double size;
```

Comment:
The variable `size` seems redundant for a `RightTriangle` since the area is calculated using `firstLeg` and `secondLeg`. Consider removing it if it's not necessary.